SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialGameInstance.h

```diff
@@ -54,6 +56,12 @@ class SPATIALGDK_API USpatialGameInstance : public UGameInstance
 	void SetFirstConnectionToSpatialOSAttempted() { bFirstConnectionToSpatialOSAttempted = true; };
 	bool GetFirstConnectionToSpatialOSAttempted() const { return bFirstConnectionToSpatialOSAttempted; };
 
+	UPROPERTY()
+	UGlobalStateManager* GlobalStateManager;
```

Comment:
Can you leave a comment why this isn't in the NetDriver anymore? As in why their lifetimes have to outlive the NetDriver.

Reply:
Added comments.